clients/p4rt_perf_test/p4rt_perf_test.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef P4RT_PERF_H
+#define P4RT_PERF_H
+
+#include <stdint.h>
+
+enum OPER { ADD = 1, DEL = 2, ADD_DEL = 3 };
```

Comment:
Rule: No dead code

You never reference ADD_DEL. You should either implement it or delete it.

Reply:
Done